src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -536,9 +551,14 @@ public void testEnrollLinesChecking() throws Exception {
 
         enrollLines = headerLine + Const.EOL + lineWithCorrectInput + Const.EOL + lineWithCorrectInputWithComment;
 
-        invalidInfo = getInvalidityInfoInEnrollLines(enrollLines, courseId);
+        invalidInfoString = null;
+        try {
+            studentsLogic.createAndValidateStudents(enrollLines, courseId);
+        } catch (EnrollException e) {
+            invalidInfoString = e.getMessage();
+        }
 
-        assertEquals(0, invalidInfo.size());
+        assertNull(invalidInfoString);
```

Comment:
In that case, it would be nice to add a comment right below it to explain why there isn't an assertion or someone might come along and "fix it"